examples/aws-cloudwatch/demographic.juttle

```diff
@@ -16,6 +16,17 @@ import "examples/aws-cloudwatch/common/aws_control_aggregate.juttle" as control_
 import "examples/aws-cloudwatch/common/aws_control_demographic.juttle" as control_demo;
 import "examples/aws-cloudwatch/common/aws_control_events.juttle" as control_events;
 
+// Take categories contributing less than x percent to the total and
+// group them together in an "Other" category.
+sub threshold(thresh) {
+    (reduce total=sum(value); pass)
+    | join
+    | put pct = 100.0*value/total
+    | put name=(pct >= thresh ? name : 'other')
+    | reduce value=sum(value), time=last(time) by name
+    | sort name
```

Comment:
sort will drop time. i expect that's ok downstream because piechart doesn't need time (even in live mode?) but it certainly undoes the `time=last(time)` above so you could skip that part.